content/cpp/concepts/polymorphism/polymorphism.md

```diff
@@ -0,0 +1,115 @@
+---
+Title: 'Polymorphism'
+Description: 'The word Polymorphism means having many form. Polymorphism is an important concept of object-oriented programming. It simply means more than one form. That is, the same entity (function or operator) can behave differently in different scenarios.'
+Subjects:
+  - 'Computer Science'
+  - 'Game Development'
+Tags:
+  - 'Function'
+  - 'Polymorphism'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+Polymorphism is an important concept of object-oriented programming. It simply means more than one form. That is, the same entity (function or operator) can operate differently under different situations.
+
+In C++ we have two types of polymorphism:
```

Comment:
```suggestion
In C++, we have two types of polymorphism:
```

Reply:
made the requested changes successfully.